plugin.video.fsgo/resources/language/English/strings.po

```diff
@@ -0,0 +1,179 @@
+﻿# Kodi FOX Sports GO language file
```

Comment:
It would be also cool if you could adjust the language folder to meet the new defaults. See here:https://github.com/xbmc/generator-kodi-addon/tree/master/generators/app/templates/resources/language